src/style.css

```diff
@@ -0,0 +1,104 @@
+
+body {
+  display: block;
+  margin: 0;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.searchBar_big_container {
+  margin: 20px 8px;
+}
+
+.searchBar_small_container {
+  margin: 0 8px 10px;
+}
+
+#searchBar_big {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  height: 70px;
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  width: 100%;
+}
+
+#searchBar_big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+#searchBar_big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir-Heavy, Arial, Helvetica, sans-serif;
+}
+
+.submit_searchButton {
+  box-sizing: border-box;
+  position: absolute;
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/
You can also set the same box-sizing for all elements using * selector